### PR TITLE
Add lease acquisition recovery interval configurability

### PR DESCRIPTION
**Pull request by duinness**
Adding the ability to modify the acquisition recovery interval because when you have a large number of streams on one account trying to recover every 5 seconds can get you stuck in a rate-limit loop.

My test environment has ~100 kinesis streams being managed by 10 instances of lifion-kinesis.
After patching my lifion-kinesis with this change and setting the leaseAquisitionRecoveryInterval to 30 seconds, errors in the environment related to this went from ~5000/min to ~20/min.

### CODE REVIEW

**Comment by duinness** [comment]
Here is the issue to which this PR should be linked
https://github.com/lifion/lifion-kinesis/issues/409

Doesn't look like I have permission to link them... or else I'm doing something wrong, which is a strong possibility 🙂

File: lib/lease-manager.js [hunk 1]
```diff
@@ -167,13 +167,16 @@ class LeaseManager {
    *        consumer or just simple polling consumers.
    * @param {number} options.leaseAcquisitionInterval - The interval in milliseconds for how often
    *        to attempt lease acquisitions.
+   * @param {number} options.leaseAcquisitionRecoveryInterval - The interval in milliseconds for how often
```
Comment (jennyEckstein) [inline lib/lease-manager.js:170]: params should be listed in abc order.  leaseAcquisitionRecoveryInterval and leaseAcquisitionInterval both should be placed in their appropriate order

Comment (jennyEckstein) [inline lib/lease-manager.js:170]: not sure why description appears broken, need make sure its ok

**Review by jennyEckstein — COMMENTED** [review]
@eaviles PR looks good to me. Just need to patch the doc

File: lib/index.js [hunk 2]
```diff
@@ -275,6 +275,8 @@ class Kinesis extends PassThrough {
    *        bucket to store large items or not.
    * @param {number} [options.leaseAcquisitionInterval=20000] - The interval in milliseconds for how often to
    *        attempt lease acquisitions.
+   * @param {number} [options.leaseAcquisitionRecoveryInterval=5000] - The interval in milliseconds for how often
```
Comment (jennyEckstein) [inline lib/index.js:278]:
```suggestion

```

File: lib/index.js [hunk 3]
```diff
@@ -275,6 +275,8 @@ class Kinesis extends PassThrough {
...
+   *        to re-attempt lease acquisitions when an error is returned from aws.
```
Comment (jennyEckstein) [inline lib/index.js:279]:
```suggestion
```

**Pull request merged**